### PR TITLE
15.700 Some new definitions

### PR DESCRIPTION
**Pull request by straccio**
Added
 VentilationDay
 VentilationNight
 OpModeCooling
 OpModeVentilation
 z1OpModeCooling
 z2OpModeCooling
 z3OpModeCooling
 z3CoolingTemp

### CODE REVIEW

**Comment by straccio** [comment]
I can't understand why the OpModeVentilation is not writeable...

**Comment by michlimek** [comment]
Hi. I test your code. VentilationDay and VentilationNight work fine. But about OpModeVentilation it don't work correct. It is connected with opmode. If I change opmode, it always change state od OpVentilationMode

File: ebusd-2.1.x/en/vaillant/15.700.csv [hunk 1]
```diff
@@ -21,6 +21,8 @@ r;w,,HydraulicScheme,system schema,,,,3600,,,UIN,,,Systemschema
 r,,WaterPressure,water pressure,,,,3900,,,pressv,,,Wasserdruck
 r;w,,SolarYieldTotal,total solar yield,,,,3D00,,,energy4,,,Solarertrag gesamt
 r;w,,YieldTotal,total environmental yield,,,,3E00,,,energy4,,,Umweltertrag gesamt
+r;w,,VentilationDay,day ventilation speed,,,,4200,,,UIN,,,Ventilation speed during day
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/15.700.csv:24]: please add all changes to the german file as well

File: ebusd-2.1.x/en/vaillant/15.700.csv [hunk 2]
```diff
@@ -51,6 +53,8 @@ r,,DisplayedOutsideTemp,outside temperature,,,,7300,,,tempv,,,Außentemperatur
 r;w,,SFMode,SF mode,,,,7400,,,sfmode,,,
 r;w,,KeyCodeforConfigMenu,code for installer menu,,,,7600,,,UIN,,,000-999; code for the installer's menues
 r;w,,OpMode,operation mode,,,,7b00,,,opmode,,,operation mode of all/configured zone(s) matching OpModeEffect
+r;w,,OpModeCooling,operation mode for cooling,,,,7c00,,,opmode,,,operation mode for cooling of all/configured zone(s)
+r;w,,OpModeVentilation,operation mode for ventilation,,,,7d00,,,opmode,,,operation mode for cooling of all/configured zone(s)
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/15.700.csv:57]: please respond to @michlimek comments for this

**Review by john30 — CHANGES_REQUESTED** [review]

File: ebusd-2.1.x/en/vaillant/15.700.csv (hunk 2 above)
Comment (straccio) [reply ebusd-2.1.x/en/vaillant/15.700.csv]: Confirmed the @michlimek version of OpModeVentilation

**Pull request closed without merging**